tests/orderly/examples/mp/mp.py

```diff
@@ -0,0 +1,24 @@
+import multiprocessing
+import random
+
+import orderly
+
+# ruff: noqa: F821
```

Comment:
This is to squash the warning about `method`?  Could you work up a more pythonic PR to fix this? Something like `orderly.parameters["method"]` or `orderly.parameters.method` that looks up parameters from a running report?

Reply:
Yeah, will do separately. I was thinking we can have the `orderly.parameters(...)` function call return the values as a dictionary.

https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5255